pkg/plugin/aggregation/run_test.go

```diff
@@ -239,6 +239,10 @@ func (cp *MockCleanupPlugin) SkipCleanup() bool {
 	return cp.skipCleanup
 }
 
+func (cp *MockCleanupPlugin) GetResultFile() string {
+	return ""
+}
```

Comment:
Just necessary to add since I added the new method to the interface.

Trivial for people who want to update the library to add a similar method.